Podcast/TapGestureRecognizer+.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TapGestureRecognizer+.swift
+//  Podcast
+//
+//  Created by Mindy Lou on 10/30/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+extension UITapGestureRecognizer {
```

Comment:
Where do we use this?

Reply:
i was trying to get it to recognize tapping on a subsection of attributed text (like a link) in a uilabel, but it's not working for multi-line labels. i can remove it for now